Movie Search/SearchController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  SearchController.swift
+//  Movie Search
+//
+//  Created by Alexey Grabik on 15/04/2020.
+//
+
+import UIKit
+
+enum SearchStatus {
+    case error(String), nothingRequested, noResult, success
+}
+
+class SearchController: NSObject {
```

Comment:
no need to inherit from NSObject

Reply:
it is needed because of perform selector method for search throttling logic